corehq/apps/groups/tests/test_groups.py

```diff
@@ -49,6 +52,31 @@ def _check_all_users(userlist):
         _check_all_users([u._id for u in group.get_static_users(is_active=False)])
         _check_all_users(group.get_static_user_ids(is_active=False))
 
+    def test_remove_users(self):
+        group = Group(domain=DOMAIN, name='group',
+                      users=[self.active_user._id, self.inactive_user._id, self.deleted_user._id])
+        group.save()
+
+        group.remove_user(self.active_user._id)
+        removal_record = UserRemoval.objects.get(group_id=group.get_id, user_id=self.active_user._id)
+        self.assertIsNotNone(removal_record)
```

Comment:
True, I just like to make it clear that we're expecting a result. It could also be in a try catch with a fail in the except but that's more lines: http://stackoverflow.com/a/4319870